sources/packages/api/src/esdc-integration/sin-validation/models/sin-validation-models.ts

```diff
@@ -0,0 +1,92 @@
+import { SINValidation } from "../../../database/entities";
+import { SINValidationFileHeader } from "../sin-validation-files/sin-validation-file-header";
+import { SINValidationFileResponse } from "../sin-validation-files/sin-validation-file-response";
+
+/**
+ * Result of a file uploaded to the SFTP.
+ */
+export interface SINValidationUploadResult {
+  generatedFile: string;
+  uploadedRecords: number;
+}
+
+/**
+ * Result of a file downloaded from the SFTP.
+ */
+export interface SINValidationResponseResult {
+  header: SINValidationFileHeader;
+  records: SINValidationFileResponse[];
+}
+
+/**
+ * Record types for a ESDC SIN Validation file.
+ */
+export enum RecordTypeCodes {
+  Header = "001",
+  Record = "002",
+  Footer = "999",
+}
+
+/**
+ * Required personal information to a
+ * ESDC SIN validation be processed.
+ */
+export interface SINValidationRecord {
+  sinValidationId: number;
+  sin: string;
+  firstName: string;
+  lastName: string;
+  birthDate: Date;
+  gender: string;
+}
+
+/**
+ * Possible known types of the SIN check status.
+ */
+export enum SINCheckStatus {
+  /**
+   * Passed electronic or manual validation.
+   */
+  Passed = "1",
```

Comment:
just curios, why the values are `string`

Reply:
It is how it is defined in the ESDC file documentation. We are just reading and saving as it is.